README.md

```diff
@@ -49,6 +49,72 @@ These widgets can be customized, added, removed and even reordered
 - Notifications (Will always be visible)
 - Label
 - Mpris (Media player controls for Spotify, Firefox, Chrome, etc...)
+- Menubar with dropdown and buttons
+- Button grid
+
+Example Config: 
+```json
+{
+  ...
+  "menubar#label": {
+    "power-buttons": {
```

Comment:
We should probably use `button|menu#AnyName` instead so that we can skip the `type` in the buttons one below